packages/edit-widgets/src/components/save-button/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useDispatch } from '@wordpress/data';
+import { useState, useCallback } from '@wordpress/element';
+
+function SaveButton() {
+	const [ isSaving, setIsSaving ] = useState( false );
+	const { saveWidgetAreas } = useDispatch( 'core/edit-widgets' );
+	const onClick = useCallback( async () => {
+		setIsSaving( true );
+		await saveWidgetAreas();
```

Comment:
I did not realize this is a thing we could be doing now... and I'm not sure how I feel about it 😅